docs/reference/commandline/plugin_rm.md

```diff
@@ -12,20 +12,22 @@ parent = "smn_cli"
 # plugin rm (experimental)
 
 ```markdown
-Usage:  docker plugin rm PLUGIN
+Usage:  docker plugin rm [OPTIONS] PLUGIN [PLUGIN...]
 
 Remove a plugin
 
 Aliases:
   rm, remove
 
 Options:
-      --help   Print usage
+      -f, --force  Force the removal of an active plugin
+      --help       Print usage
 ```
 
 Removes a plugin. You cannot remove a plugin if it is active, you must disable
 a plugin using the [`docker plugin disable`](plugin_disable.md) before removing
-it.
+it (or use --force, use of force is not recommended, since it can affect
```

Comment:
Currently, containers using a plugin are not reference counted. Once that's implemented, a plugin with refCount > 0 cannot be disabled, except for when it is force disabled. To summarize:
1. plugin disable --force: should disable plugin, even if there are referencing user containers.
2. plugin remove --force: should remove plugin, after disabling plugin. Use disable --force, if necessary.